packages/core/src/components/widgets/AltDateWidget.tsx

```diff
@@ -41,13 +47,16 @@ function DateElement(props) {
       disabled={disabled}
       readonly={readonly}
       autofocus={autofocus}
-      onChange={(value) => select(type, value)}
+      onChange={(value: any) => select(type, value)}
       onBlur={onBlur}
     />
   );
 }
 
-class AltDateWidget extends Component {
```

Comment:
Can you add JSDoc here as well... take a look at how the other, already converted widgets are documented as a guide. Bonus points if you document all of the components and functions in the entire file!